### PR TITLE
Side 2091 export ldk

### PR DESCRIPTION
**Pull request by cshields-oliveai**

### CODE REVIEW

File: ldk/goja/src/filesystem/index.ts [hunk 1]
```diff
@@ -267,7 +267,7 @@ export function writeFile(
   });
 }
 
-export function join(segments: [string]): Promise<string> {
+export function join(segments: string[]): Promise<string> {
```
Comment (cshields-oliveai) [inline ldk/goja/src/filesystem/index.ts:270]: typo here in the join - was sending in a string tuple instead of a string[].

**Review by RusGraf — APPROVED** [review]
image

File: ldk/goja/src/index.ts [hunk 2]
```diff
@@ -6,7 +6,31 @@ import * as network from './network';
...
+const _process: process.Process = process;
+const _ui: ui.UI = ui;
+const _vault: vault.Vault = vault;
+const _whisper: whisper.WhisperAptitude = whisper;
+const _window: window.Window = window;
+/*eslint-disable */
 
-export { clipboard, cursor, filesystem, keyboard, network, process, ui, vault, window, whisper };
+export {
+    clipboard,
```
Comment (scottkipferOlive) [inline ldk/goja/src/index.ts:26]: won't this till have the clipboard.clipboard problem?

Comment (cshields-oliveai) [reply ldk/goja/src/index.ts:26]: @scottkipferOlive - We have tested this out in a few projects, and are seeing the expected behavior. I've attached a few screenshots illustrating how you can import these aptitudes.

Screen Shot 2021-04-22 at 11 15 02 AM
Screen Shot 2021-04-22 at 11 14 48 AM
Screen Shot 2021-04-22 at 11 17 43 AM

Is there something else we are missing?

File: ldk/goja/src/index.ts [hunk 3]
```diff
@@ -6,7 +6,31 @@ import * as network from './network';
...
+const _cursor: cursor.Cursor = cursor;
+const _keyboard: keyboard.Keyboard = keyboard;
+const _filesystem: filesystem.Filesystem = filesystem;
+const _network: network.Network = network;
+const _process: process.Process = process;
+const _ui: ui.UI = ui;
+const _vault: vault.Vault = vault;
+const _whisper: whisper.WhisperAptitude = whisper;
+const _window: window.Window = window;
+/*eslint-disable */
```
Comment (richseviora) [inline ldk/goja/src/index.ts:23]:
```suggestion
/* eslint-enable */
```

**Review by richseviora — CHANGES_REQUESTED** [review]
One minor change - otherwise LGTM :)

File: ldk/goja/src/index.ts (hunk 2 above)
Comment (scottkipferOlive) [reply ldk/goja/src/index.ts:26]: Looks good

**Comment by cshields-oliveai** [comment]
@richseviora I need another approval on your requested changes.

**Review by richseviora — APPROVED** [review]

**Pull request merged**